homeassistant/components/mqtt/quality_scale.yaml

```diff
@@ -0,0 +1,131 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: |
+      Service actions are set up in `async_setup_entry` and are only set up if the MQTT broker was set up correctly.
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions:
+    status: done
+    comment: |
+      The integration can be removed by removing the config entry.
```

Comment:
Yes but is this mentioned in the docs?

Reply:
Is this needed as it is a standard to setup an integration though the UI using a config flow. May be it would be better to update the `Manual configurations steps` template we insert in the docs instead and all it `Manual configuration and removal instructions`?

Reply:
https://github.com/home-assistant/home-assistant.io/pull/35875

Reply:
Ready when PR is merged, comment was removed.

Reply:
Docs PR was merged